requirements/app/docs.txt

```diff
@@ -8,7 +8,7 @@ docutils>=0.16
 sphinxcontrib-fulltoc>=1.0
 sphinxcontrib-mockautodoc
 
-https://storage.googleapis.com/grid-packages/lightning-ai-sphinx-theme/build-31-rc1.zip
+https://storage.googleapis.com/grid-packages/lightning-ai-sphinx-theme/build-31.3.zip
```

Comment:
@yurijmikhalevich Why not have it as an installable repo?

Reply:
@kaushikb11, I think it's a good idea for the future. I just copied what we did with the `pytorch` docs theme because it's faster: https://github.com/Lightning-AI/lightning/blob/91dd6a68fb596d45914fc5d4fbbf2bad52e8399e/requirements/pytorch/docs.txt#L9.